config/initializers/zeitwerk.rb

```diff
@@ -0,0 +1,6 @@
+Rails.autoloaders.each do |autoloader|
+  autoloader.inflector.inflect(
+    "csv_presenter" => "CSVPresenter",
+    "csv_parser" => "CSVParser",
+  )
+end
```

Comment:
I'd add those to the general inflections, and not just to the autoloader, like this:
```ruby
# config/initializers/inflections.rb
ActiveSupport::Inflector.inflections(:en) do |inflect|
  inflect.acronym "CSV"
end
```
as it's more general. I'd only avoid that if it was something likely to clash with something else.

Reply:
It clashes with other things - there's CSV and Csv used